vms/rpcchainvm/vm_client.go

```diff
@@ -353,8 +353,10 @@ func (vm *VMClient) SetState(ctx context.Context, state snow.State) error {
 
 func (vm *VMClient) Shutdown(ctx context.Context) error {
 	errs := wrappers.Errs{}
-	_, err := vm.client.Shutdown(ctx, &emptypb.Empty{})
-	errs.Add(err)
+	if vm.processTracker.HasProcess(vm.pid) {
+		_, err := vm.client.Shutdown(ctx, &emptypb.Empty{})
+		errs.Add(err)
+	}
```

Comment:
If the process doesn't exist - I think that should be reported as a (fatal) error here.

Reply:
fail fast makes sense